### PR TITLE
[IMP] l10n_hr : Croatian localization

### PR DESCRIPTION
**Pull request by malb-odoo**
Improving:
- Chart of account
- Tax report
- Fiscal position
- Translation

Task-id: 2753362

Signed-off-by: Maximilien La Barre <malb@odoo.com>
--
I confirm I have signed the CLA and read the PR guidelines at [www.odoo.com/submit-pr](http://www.odoo.com/submit-pr)

### CODE REVIEW

File: addons/l10n_hr/data/account.tax.group.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 id,name,country_id/id
-tax_group_0,PDV 0%,base.hr
-tax_group_10,PDV 10%,base.hr
-tax_group_25,PDV 25%,base.hr
-tax_group_30,PDV 30%,base.hr
-tax_group_70,PDV 70%,base.hr
```
Comment (william-andre) [inline addons/l10n_hr/data/account.tax.group.csv:6]: Wtf :open_mouth:

File: addons/l10n_hr/data/account.group.template.csv [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+hr_group_83,83,,"Profit or loss for the year attributable to others",l10n_hr.l10n_hr_chart_template
+hr_group_9,9,,"Equity and reserves and of the balance sheet items",l10n_hr.l10n_hr_chart_template
+hr_group_90,90,,"Share capital",l10n_hr.l10n_hr_chart_template
+hr_group_91,91,,"Capital reserves",l10n_hr.l10n_hr_chart_template
+hr_group_92,92,,"Reserves from retained earnings",l10n_hr.l10n_hr_chart_template
+hr_group_93,93,,"Revaluation reserves and fair value reserves",l10n_hr.l10n_hr_chart_template
+hr_group_94,94,,"Retained earnings or accumulated loss",l10n_hr.l10n_hr_chart_template
+hr_group_95,95,,"Profit or loss for the year",l10n_hr.l10n_hr_chart_template
+hr_group_96,96,,"Non-controlling interests",l10n_hr.l10n_hr_chart_template
+hr_group_99,99,,"Off-balance sheet balance",l10n_hr.l10n_hr_chart_template
```
Comment (william-andre) [inline addons/l10n_hr/data/account.group.template.csv:90]: eol

**Comment by badbole** [comment]
Hello .. i belive this PR is related to mine done just a bit before...

#102018 - Improoved CoA, Tax rport lines and taxes...

I can make external ids change to follow common logic and prefix then with l10n_hr
but i belive it is more adequate and covering more taxes compared to this one.. 

It is also done with old filed instact for easier comparison... 

Please let me know should i modify my PR or will you continue with your modification?

**Comment by william-andre** [comment]
@badbole We will proceed with this PR, if there are some remarks or changes needed, please add a commit on top of this one. It will make the review easier, thank you!

**Comment by badbole** [comment]
@william-andre this PR has taxes setup wrong, no fiscal positions.. 
if you want correct values for localisation.. i made a PR ofr it... 
will correct all translations and do needed modifications... but only in my branch ... 

i made all comparisons and explained the diff of 
account charts and account groups from this branch compared to my previous attempt
in this comment: https://github.com/odoo/odoo/pull/106485#issuecomment-1327171622

so if you think this PR is compliant for usage in croatia..... 
fine.. proceed with it... guess you know things better

File: addons/l10n_hr/data/fiscal_templates_data.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <field name="vat_required" eval="True"/>
+            <field name="country_id" ref="base.hr"/>
+        </record>
+        <record id="fiscal_position_hr_person_private" model="account.fiscal.position.template">
+            <field name="name">Partner private</field>
+            <field name="sequence">20</field>
+            <field name="chart_template_id" ref="l10n_hr_chart_template"/>
+            <field name="auto_apply" eval="True"/>
+            <field name="country_group_id" ref="base.europe"/>
+        </record>
```
[the remark this answers is not in the document]
Comment (badbole) [reply addons/l10n_hr/data/fiscal_templates_data.xml]: wrong usage...  based on google translate.

File: addons/l10n_hr/i18n/hr.po [hunk 4]
```diff
@@ -0,0 +1,5477 @@
...
+#: model:account.account.template,name:l10n_hr.hr_620
+msgid "Biological assets in progress"
+msgstr "Biološka imovina u izradi"
+
+#. module: l10n_hr
+#: model:account.account,name:l10n_hr.1_hr_040
+#: model:account.account,name:l10n_hr.2_hr_040
+#: model:account.account.template,name:l10n_hr.hr_040
+msgid "Biological assets- growing crops"
+msgstr "Biološka imovina - uzgoj usjeva"
```
[the remark this answers is not in the document]
Comment (malb-odoo) [reply addons/l10n_hr/i18n/hr.po:1019]: I think at the time i just used google translate but i can change the po to match the official document indeed. Can't remember why i did that, i will change this soon

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**